### DISCUSSION TITLE
Does it support DFR0768?

### DISCUSSION THREAD

**Discussion by SanderM2** [Makuna/DFMiniMp3#134] [Q&A]
Does this library support the DFR0768 (DFPlayer Pro)?
I'm not sure if the protocol is the same as the mini, but I guess so?

**Comment by Makuna** [comment]
Not it does not support the PRO.  They use a completely different protocol.  

The new PRO uses an old AT command protocol (remember acoustic modems, they used this command protocol).  The AT commands still live on still but often only communications (like RF transceivers in our phones).
